modules/internal/ChapelBase.chpl

```diff
@@ -3377,7 +3377,7 @@ module ChapelBase {
     if zippered && isTuple(iterable) then
       return chpl_boundedCoforallSize(iterable[0], zippered=false);
     else if isRange(iterable) || isDomain(iterable) || isArray(iterable) then
-      return iterable.sizeAs(iterable.intIdxType);
+      return iterable.size;
```

Comment:
This is an internal module, is there a reason this one became `.size` instead of `.sizeAs(iterable.chpl_integralIdxType`?

Reply:
My rationale for this was:
* This is computing the number of iterations of a coforall whose bounds are well-defined, and it seems unlikely that any actual coforall has more than max(int) iterations in practice (since that would create a lot of tasks / require a lot of cores and memory to actually work correctly)
* It wouldn't surprise me if, in practice, we had never tested code paths for this that used sizes other than `int`.  Specifically, I feel a little skeptical that something reasonable would happen if `intIdxType` had been `uint`
* It seemed much simpler / more straightforward as an implementation, and to say that this routine returns a default `int`
* It didn't break any testing